problemsets-mat223/linearalbegra.tex

```diff
@@ -1561,9 +1953,23 @@ \section*{Dot Product}
 	Let $\vec u=\mat{1\\2\\1}$ and $\vec v=\mat{1\\1\\3}$.
 	\begin{parts}
 		\item Find the distance between $\vec u$ and $\vec v$.
+			\begin{solution}
+				$\vec u - \vec v = \mat{0\\1\\-2}$, and so $\|\vec u-\vec v\| = \sqrt{5}$. 
+			\end{solution}
 		\item Find a unit vector in the direction of $\vec u$.
+			\begin{solution}
+				$\frac{1}{\sqrt{6}} \vec u = \mat{\frac{1}{\sqrt{6}}\\\frac{2}{\sqrt{6}}\\\frac{1}{\sqrt{6}}}$.
+
+				$\|\vec u\| = \sqrt{6}$, and so if we multiply $\vec u$ by $\frac{1}{\sqrt{6}}$,
+				the length of the resulting vector will be 1. 
+			\end{solution}
 		\item Does there exists a \emph{unit vector} $\vec x$ that is distance
 			$1$ from $\vec u$?
+			\begin{solution}
+				No. $\|\vec u\| = \sqrt{6}$, and so the shortest length that a
+				vector whose distance from $\vec u$ is 1 can have is $\sqrt{6} - 1$, 
+				which is greater than 1. 
+			\end{solution}
 		\item Suppose $\vec y$ is a unit vector and the distance between $\vec y$ and
 			$\vec u$ is $2$.  What is the angle between $\vec y$ and $\vec u$?
```

Comment:
Quick question about this problem (problem 21.4, line 1973). The solution to this I imagine requires understanding the distributivity of the dot product: 

 4 = (u - v) \cdot (u - v) = u \cdot u - 2 u \cdot v + v \cdot v

(Then use that to compute u \cdot v, from which you can get the angle.)

That isn't something that is explicitly mentioned earlier. Is that something we expect them to discover? If so, to what extent should I be explaining it before using it? 

Or am I just totally missing some more elementary way of doing this?

Reply:
Yeah, they will know distributivity of the dot product. The unfortunate state of affairs is that for most students it won't even occur to them to ask, "hey, is this new product distributive?" since to them all things are distributive. We don't emphasize axioms in this course, so I'm okay with them just treating the dot product as distributive without proving it or having it as part of the definition.